app/src/ui/acknowledgements/acknowledgements.tsx

```diff
@@ -34,10 +34,21 @@ export class Acknowledgements extends React.Component<IAcknowledgementsProps, IA
   public constructor(props: IAcknowledgementsProps) {
     super(props)
 
-    this.state = { licenses: null }
+    this.state = { licenses: null, licenseText: null }
   }
 
   public componentDidMount() {
+    const licensePath = Path.join(getAppPath(), 'static', 'LICENSE')
```

Comment:
LOL, actually this suffers from the same problem as mentioned here https://github.com/desktop/desktop/issues/1576#issuecomment-302363884

<img width="345" src="https://cloud.githubusercontent.com/assets/359239/26226942/a637897e-3c72-11e7-8995-6c4ca9e574c6.png">